src/util/web-util.js

```diff
@@ -52,14 +52,14 @@ function composePatchQuery (toDel, toIns) {
     toDel = toDel.map(function (st) {
       return statementToNT(st, excludeDot)
```

Comment:
What's the point of calling `excludeDot` every time? All NT statements end with a dot.